src/components/RevokeContainer/RevokeTag/RevokeTag.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from "@testing-library/react";
+import { RevokeTag } from "./RevokeTag";
+import { BrowserRouter } from "react-router-dom";
+import sampleRevokableDocument from "../../../test/fixtures/sample-wrapped-document.json";
+
+describe("RevokeTag", () => {
+  it("should render the file tag correctly with then given properties", () => {
+    render(
```

Comment:
could add in 1 more control test for 0 file size check

Reply:
dont understand this, why do we need to test for 0 file size check?